README.md

```diff
@@ -99,6 +99,14 @@ The prompt shows information you need while you're working, while staying sleek
    $ cargo install starship
    ```
 
+   #### Arch Linux (AUR)
+
+   Starship is available on the AUR under the package name `starship`. Install it with `yay` or your favorite AUR helper.
+
+   ```
+   $ sudo yay -S starship
```

Comment:
Conventionally AUR helper works without sudo they themselves asked privileged sudo access. It is believe to be bad habit of using sudo with AUR helpers.

Reply:
You know what...I had forgotten about that.

My usual helper only escalates to sudo when needed for dependency installs, but you're right.